assets/js/cookieBanner.js

```diff
@@ -21,10 +21,22 @@
   }
 
   function hideCookiesConfirmation(banner) {
-    banner.querySelectorAll('#cookie-banner-accepted, #cookie-banner-rejected')
-      .forEach(message => message.hidden = true)
+    banner
+      .querySelectorAll('#cookie-banner-accepted, #cookie-banner-rejected')
+      .forEach(message => (message.hidden = true))
 
-      banner.hidden = true
+    banner.hidden = true
+  }
+
+  function removeAnalyticsCookies() {
+    const domain = location.hostname === 'localhost' ? 'localhost' : 'justice.gov.uk'
```

Comment:
By default, GA seems to set cookies for `justice.gov.uk` - unless it's localhost....

Reply:
Is it possible to configure this in GA? I wonder if it's our account is set to cover all that domain? It just means the GA cookie is accessible from any site on that domain I think?

Reply:
Yeah, that's right - seems it's a Google default to go for a fairly high-level sub-domain. Most other HMPPS services seem to be leaving the default set as far as I can tell because you accumulate a lot of `_ga_[containerId]` if you do a bit of browsing around DPS and linked services. One Login does similar on their domains too. I guess it enables tracking journeys across DPS services, for example 🤷‍♂️ ?

I'll leave it as it is for now (as you've approved it) but will raise on the ticket to confirm whether we should be setting a more specific GA cookie domain.